testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/ClientAuthSignedJWTTest.java

```diff
@@ -845,7 +845,9 @@ protected JsonWebToken createRequestToken(String clientId, String realmInfoUrl)
 
             int now = Time.currentTime();
             if (isClaimEnabled("issuedAt")) reqToken.issuedAt(now);
-            if (isClaimEnabled("expiration")) reqToken.expiration(now + getTokenTimeout());
+            // For the time being there's no getter for tokenTimeout in JWTClientCredentialsProvider
```

Comment:
As CustomJWTClientCredentialsProvider extends the JWTClientCredentialsProvider you could just override the setTokenTimeout to make the tokenTimeout available to this test.